src/style.css

```diff
@@ -1 +1,91 @@
-/* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  height: 16px;
+  width: 24px;
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-right: 4px;
+}
+
+.stars--1 :nth-child(1) {
+  background-image: url(images/star-active.svg);
+  background-repeat: no-repeat;
+}
+
+.stars--2 :nth-child(1) {
+  background-image: url(images/star-active.svg);
+  background-repeat: no-repeat;
+}
+
+.stars--2 :nth-child(2) {
+  background-image: url(images/star-active.svg);
+  background-repeat: no-repeat;
+}
+
+.stars--3 :nth-child(1) {
+  background-image: url(images/star-active.svg);
+  background-repeat: no-repeat;
+}
+
+.stars--3 :nth-child(2) {
+  background-image: url(images/star-active.svg);
+  background-repeat: no-repeat;
+}
+
+.stars--3 :nth-child(3) {
+  background-image: url(images/star-active.svg);
+  background-repeat: no-repeat;
+}
+
+.stars--4 :nth-child(1) {
+  background-image: url(images/star-active.svg);
+  background-repeat: no-repeat;
+}
+
+.stars--4 :nth-child(2) {
+  background-image: url(images/star-active.svg);
+  background-repeat: no-repeat;
+}
+
+.stars--4 :nth-child(3) {
+  background-image: url(images/star-active.svg);
+  background-repeat: no-repeat;
+}
+
+.stars--4 :nth-child(4) {
+  background-image: url(images/star-active.svg);
+  background-repeat: no-repeat;
+}
+
+.stars--5 :nth-child(1) {
+  background-image: url(images/star-active.svg);
+  background-repeat: no-repeat;
+}
+
+.stars--5 :nth-child(2) {
+  background-image: url(images/star-active.svg);
+  background-repeat: no-repeat;
+}
+
+.stars--5 :nth-child(3) {
+  background-image: url(images/star-active.svg);
+  background-repeat: no-repeat;
+}
+
+.stars--5 :nth-child(4) {
+  background-image: url(images/star-active.svg);
+  background-repeat: no-repeat;
+}
+
+.stars--5 :nth-child(5) {
+  background-image: url(images/star-active.svg);
+  background-repeat: no-repeat;
+}
```

Comment:
```suggestion
.stars--1 :nth-child(-n + 1),
.stars--1 :nth-child(-n + 2), 
.stars--1 :nth-child(-n + 3), 
.stars--1 :nth-child(-n + 4), 
.stars--1 :nth-child(-n + 5),  {
  background-image: url(images/star-active.svg);
}
```
it is can be much shorter.
and no need to set again `background-repeat: no-repeat;`
it is already settled in .stars_star selector

Reply:
![image](https://github.com/mate-academy/layout_stars/assets/157310416/97508ddf-dd33-4390-9a06-bcbc7d9aafdc)

I dont know why, but i cant create a new commit

Reply:
but I take your point

Reply:
If you have any questions or problems - feel free to ask them in fe_chat